src/V3OrderProcessDomains.cpp

```diff
@@ -93,6 +93,13 @@ class V3OrderProcessDomains final {
             // For logic, start with the explicit hybrid sensitivities
             OrderLogicVertex* const lvtxp = vtxp->cast<OrderLogicVertex>();
             if (lvtxp) domainp = lvtxp->hybridp();
+            if (domainp)
+                UINFO(6, "      hybr d=" << cvtToHex(domainp)
+                                         << (domainp == m_deleteDomainp ? " [DEL]"
+                                             : domainp->hasCombo()      ? " [COMB]"
+                                             : domainp->isMulti()       ? " [MULT]"
+                                                                        : "")
```

Comment:
Similar code here in three places - add a common function?  Or method to the appropriate class.